lib/records/fhir_dstu2.rb

```diff
@@ -332,6 +353,29 @@ def self.observation(observation, fhir_record, patient, encounter)
         fhir_record.entry << entry
       end
 
+      def self.quality_of_life_observation(observation, fhir_record, patient)
```

Comment:
Except the names of the fields are different in the versions, with different cardinalities sometimes. So, that is a big problem.